projects/frontend/data-pipelines/gui/projects/data-pipelines/src/lib/components/base-grid/_data-jobs-base-grid.component.scss

```diff
@@ -1,3 +1,8 @@
+/*!
+ * Copyright 2023-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
 /*
  * Copyright 2021-2023 VMware, Inc.
  * SPDX-License-Identifier: Apache-2.0
```

Comment:
Please cleanup from duplicates?

Reply:
Seems to be due the license pre-commit hook changes made in the scope of this PR to me

Reply:
Yes exactly, it's because pre-commit config changes from my side and they pop-up from the other library data-pipelines, but local pre-commit didn't catch them because I didn't have changes there

Reply:
I fix them and there is amend to existing commit and squash everything into 1 commit